shared-libs/contact-types-utils/test/index.js

```diff
@@ -155,6 +155,21 @@ describe('ContactType Utils', () => {
     });
   });
 
+  describe('isSameContactType', () => {
+    it('should return true for contacts of the same type', () => {
+      chai.expect(utils.isSameContactType([
+        { type: 'contact', contact_type: 'health_center' },
+        { type: 'contact', contact_type: 'health_center' },
+      ])).to.equal(true);
+    });
+    it('should return false for contacts of different type', () => {
+      chai.expect(utils.isSameContactType([
+        { type: 'contact', contact_type: 'health_center' },
+        { type: 'contact', contact_type: 'district_hospital' },
+      ])).to.equal(false);
```

Comment:
Can we have a test where we use a mix of hardcoded and configurable hierarchies? so so using both only `type` and both `type` and `contact_type` together.